tests/test_path.py

```diff
@@ -356,6 +356,7 @@ def test_root_name(self, alpharep):
         """
         The name of the root should be the name of the zipfile
         """
+        alpharep = self.zipfile_ondisk(alpharep)
```

Comment:
I'd prefer to have tests that explicitly test the missed expectation instead of incidentally capturing it because an on-disk representation happened to be at play. Also, this change removes the assertions about zip files not on disk. Let's extract a separate test (or tests).

I'll work on it, since I think we can just knock it out.